include/functions.h

```diff
@@ -1885,7 +1885,7 @@ void UCodeDisas_Destroy(UCodeDisas*);
 // ? UCodeDisas_ParseGeometryMode(?);
 // ? UCodeDisas_ParseRenderMode(?);
 // ? UCodeDisas_PrintVertices(?);
-void UCodeDisas_Disassemble(UCodeDisas*, Gfx*);
+// void UCodeDisas_Disassemble(UCodeDisas*, Gfx*);
```

Comment:
we should remove the comments for any prototypes of functions that are not meant to be called outside of the system

Reply:
`UCodeDisas_Disassemble` is used in `graph.c` here
https://github.com/zeldaret/oot/blob/ceb571423db37b87336bff905078aabc6c627572/src/code/graph.c#L44
and here
https://github.com/zeldaret/oot/blob/ceb571423db37b87336bff905078aabc6c627572/src/code/graph.c#L78

Reply:
alright, then we should remove the //